src/helpers/Blocks/Blocks.js

```diff
@@ -152,44 +152,32 @@ export function addBlock(formData, type, index) {
 }
 
 export function mutateBlock(formData, id, value) {
```

Comment:
Where are we calling this method?

Reply:
`mutateBlock` in not called in the core, but maybe some add-on rely on it. If we remove it, I think we need to have a breaking release.

@sneridagh ^